packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -134,29 +171,51 @@ export const prepareLocalChainAccountKit = (
         async makeDelegateInvitation(validatorAddress, ertpAmount) {
           trace('makeDelegateInvitation', validatorAddress, ertpAmount);
 
+          return zcf.makeInvitation(async seat => {
+            // TODO should it allow delegating more BLD?
+            seat.exit();
+            return this.facets.holder.delegate(validatorAddress, ertpAmount);
+          }, 'Delegate');
+        },
+        /**
+         *
+         * @param {string} validatorAddress
+         * @param {Amount<'nat'>} ertpAmount
+         */
+        async undelegate(validatorAddress, ertpAmount) {
           // TODO #9211 lookup denom from brand
           const amount = {
             amount: String(ertpAmount.value),
             denom: 'ubld',
           };
+          const { account: lca } = this.state;
+          trace('lca', lca);
+          const delegatorAddress = await E(lca).getAddress();
+          trace('delegatorAddress', delegatorAddress);
+          const [result] = await E(lca).executeTx([
+            typedJson('/cosmos.staking.v1beta1.MsgUndelegate', {
+              amount,
+              validatorAddress,
+              delegatorAddress,
+            }),
+          ]);
+          trace('got result', result);
+          return result;
+        },
+
+        /**
+         *
+         * @param {string} validatorAddress
+         * @param {Amount<'nat'>} ertpAmount
+         */
+        async makeUndelegateInvitation(validatorAddress, ertpAmount) {
```

Comment:
I'd be inclined to inlined this into the InvitationMakers facet.

Reply:
I vacillated on that. I thought maybe holders of the other facet should be able to make these invitations too. But I suppose they're entirely empowered to do so by open coding. I'll take it off this facet